site_scons/env_modules.py

```diff
@@ -168,9 +168,9 @@ def show_avail(self):
             print("Could not invoke module avail")
         return output
 
-    def get_map(self):
+    def get_map(self, key):
```

Comment:
Not sure this is related to the PR but looks a good change.

Reply:
It's a bug-fix.